modules/pyramid/src/pyramid.cpp

```diff
@@ -0,0 +1,321 @@
+#include "precomp.hpp"
+
+using namespace std;
+using namespace cv;
```

Comment:
Thanks for your comment.
I think that I didn't understand your comment. I have tried  
`namespace cv
{
namespace pyramid
{

/* Reference
This code is patened http://www.google.com/patents/US20140072229
1 ICCP 2014 Riesz Pyramids for Fast Phase-Based Video Magnification  http://people.csail.mit.edu/nwadhwa/riesz-pyramid/
2 In "Supplemental for Riesz Pyramids for Fast Phase-Based Video Magnification" supplemental material : http://people.csail.mit.edu/nwadhwa/riesz-pyramid/RieszPyrSupplemental.zip


*/

Pyramid::Pyramid(Pyramid const &p)
{
`
but there is still a link error. I have got VS2013 and 2015 and there is no error. I don't know how to reproduce this error. Don't you know an option in VS to have same error?